lib/command.js

```diff
@@ -348,9 +366,7 @@ module.exports = function command (yargs, usage, validation, globalMiddleware) {
         unparsed.push(value)
       })
     })
-
-    // short-circuit parse.
-    if (!unparsed.length) return
+    if (!unparsed.length) return detailedPositionalParse
```

Comment:
In that case, postProcessPositionals (now runParserOnPositionals) used to return nothing and populatePositionals (now addPositionalsToArgv) used to return positionalMap.

Now, runParserOnPositionals returns `{}` and addPositionalsToArgv returns it too. Isn't something missing? Shouldn't detailedPositionalParse contain at least values from positionalMap before being returned?